astropy/table/operations.py

```diff
@@ -660,8 +661,8 @@ def hstack(tables, join_type='outer',
 
     Parameters
     ----------
-    tables : Table or list of Table or Row objects
-        Tables to stack along columns (horizontally)
+    tables : list of :class:`~astropy.table.Table` or :class:`~astropy.table.Row`
```

Comment:
This can also be a single table.

Reply:
done. The docstring is a little long, but I don't think it's throwing any style errors.

Reply:
✅